src/main/java/com/wanderersoftherift/wotr/init/ModCreativeTabs.java

```diff
@@ -26,7 +26,12 @@ public class ModCreativeTabs {
                         output.accept(ModItems.SHAPED_RUNEGEM_GEODE);
                         output.accept(ModItems.CUT_RUNEGEM_GEODE);
                         output.accept(ModItems.POLISHED_RUNEGEM_GEODE);
-                        output.accept(ModItems.FRAMED_RUNEGEM_GEODE);
+                        output.accept(ModItems.FRAMED_RUNEGEM_MONSTER);
```

Comment:
I believe this should be FRAMED_RUNEGEM_GEODE (there's another FRAMED_RUNEGEM_MONSTER below)